selfdrive/statsd.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+import os
+import zmq
+import json
+import time
+from pathlib import Path
+from collections import deque
+from datetime import datetime
+from selfdrive.hardware import PC
+from cereal.messaging import SubMaster
+from common.file_helpers import atomic_write_on_fs_tmp
+
+
+AGGREGATION_TIME_S = 60
+STATS_SOCKET = "ipc:///tmp/stats"
+if PC:
```

Comment:
Maybe put this logic in `selfdrive.loggerd.config`?

Reply:
yeah, let's keep all this stuff in one place